packages/api/src/routes/nfts-upload.js

```diff
@@ -132,7 +132,7 @@ export async function uploadCar(params) {
 }
 
 /**
- * @param {UploadCarInput} data
+ * @param {Omit<UploadCarInput, 'car'>} data
```

Comment:
the `car` property is no longer needed by `uploadCarWithStat` as `CarStat` now provides the `carBytes` as a `UInt8Array`